### PR TITLE
UG changes, part 1

### PR DESCRIPTION
**Pull request by zhongfu**
might be easier to review commit-by-commit

feel free to reject any part of it

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -9,25 +9,25 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
 You can input your employees' data and the jobs that you want to keep track of.
 Then, you can assign the employees to the jobs that they are working on.
 After the job is completed, you can mark the job as paid, and PeopleSoft will calculate how much each employee is to be paid based on their hourly rates.
-You can also generate a payslip in CSV format for your employees to refer to.
+You can also generate a payslip in comma-separated values (CSV) format for you and your employees to refer to.
 
 The program simulates a real life workflow:
+  * Company receives a new job.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:15]:
```suggestion
  * The company receives a new job.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
...
 
 **:information_source: How to use this guide:**<br>
 
 * Words in `monospace font` are commands to be typed into PeopleSoft.<br>
 
-* Words in `UPPERCASE` are to be replaced with an appropriate value.<br>
-  e.g. `n/NAME` means that `NAME` should be substituted with a name, e.g. `n/John Doe`.
+* Boxes like these (with a blue background and the :information_source: icon) contain relevant tips for using PeopleSoft.
+
+* Boxes with a yellow background and the :warning: icon contain important warnings.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:29]: Do the yellow backgrounds work again? I didn't see them in my previous review on the website

File: docs/UserGuide.md [hunk 3]
```diff
@@ -39,268 +39,292 @@ The program simulates a real life workflow:
...
 
 2. Hit **enter** to run the command.
 
-A new job should be created in the table of jobs. It should have the name 'Repair aircon' with a duration of '2h'.
+    A new job should be created in the table of jobs. It should have the name 'Repair aircon' with a duration of '2h'.
 
 #### Assign person to a job
 
-Like in the real-world, jobs need to be assigned to employees. This can be done in PeopleSoft with the command `assign`. `assign` is a command that is **index-based**, this means that it uses the ordering of the items displayed in the table. You will have to specify at least two indexes, one for the job, and at least one for the employees.
+PeopleSoft allows you to indicate which employees are in-charge of a certain job with the `assign` command. The `assign` command requires 1 job *index*, and at laest 1 employee *index* (each prefixed by `i/`). These indexes can be found under the `#` column in both tables.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:91]: little typo
```suggestion
PeopleSoft allows you to indicate which employees are in-charge of a certain job with the `assign` command. The `assign` command requires 1 job *index*, and at least 1 employee *index* (each prefixed by `i/`). These indexes can be found under the `#` column in both tables.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -9,25 +9,25 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
...
 * Words in `monospace font` are commands to be typed into PeopleSoft.<br>
 
-* Words in `UPPERCASE` are to be replaced with an appropriate value.<br>
-  e.g. `n/NAME` means that `NAME` should be substituted with a name, e.g. `n/John Doe`.
+* Boxes like these (with a blue background and the :information_source: icon) contain relevant tips for using PeopleSoft.
+
+* Boxes with a yellow background and the :warning: icon contain important warnings.
 
-* Words in square brackets are optional.<br>
-  e.g. `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:30]: Was this block moved elsewhere in the document? Can't seem to find it. Or did you reword such that this sentence is not needed at all?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -39,268 +39,292 @@ The program simulates a real life workflow:
...
 Format: `persondelete PERSON_INDEX`
 
-Example: `persondelete 3` deletes the third person in the list
+Example: `persondelete 3` deletes the third person in the list.
 
-#### How to search for a person by name or tag? `personfind`
+#### `personfind`: Find employees by name or tag
 
-Finds all people by a certain name and/or tag. 
+Finds all employees that have the given keyword(s) in their names or tags, and lists them in the employee table.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:289]: I used 'employee list' in the labelled UI screenshot above so they might be confused if it is called table now
```suggestion
Finds all employees that have the given keyword(s) in their names or tags, and lists them in the employee list.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -39,268 +39,292 @@ The program simulates a real life workflow:
...
 
-If multiple tags are entered, only entries that match **all** tags are returned.
+If multiple keywords are entered, only entries that match **all** keywords are returned.
```
Comment (ian-from-dover) [inline docs/UserGuide.md:291]: Good change 👍

File: docs/UserGuide.md [hunk 7]
```diff
@@ -388,101 +425,142 @@ assigned to a job, un-mark the job first before assigning them.
...
+
+Opens the help page, which includes a list of commands, command formats, and example usages.
 
 Format: `help`
 
-![help](images/screenshots/help/help.png)
+Example: `help` opens up the help page.
 
-Prompted when the user makes a typo (i.e. tries to use an invalid command):
-Format: `Looks like you used an invalid command. Use the command help to access a list of all available commands.`
```
Comment (ian-from-dover) [inline docs/UserGuide.md:449]: Good remove. I felt this was redundant too

**Review by ian-from-dover — APPROVED** [review]
Overall good edits. Remember to fix the CI

File: docs/UserGuide.md [hunk 8]
```diff
@@ -9,25 +9,25 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
 You can input your employees' data and the jobs that you want to keep track of.
 Then, you can assign the employees to the jobs that they are working on.
 After the job is completed, you can mark the job as paid, and PeopleSoft will calculate how much each employee is to be paid based on their hourly rates.
-You can also generate a payslip in CSV format for your employees to refer to.
+You can also generate a payslip in comma-separated values (CSV) format for you and your employees to refer to.
 
 The program simulates a real life workflow:
+  * Company receives a new job.
+  * The HR manager `add`s the job to PeopleSoft, and `assign`s employees to work on it.
   * Employees start working.
```
Comment (zhongfu) [inline docs/UserGuide.md:17]:
```suggestion
  * The employees start working. 
```

File: docs/UserGuide.md (hunk 4 above)
Comment (zhongfu) [reply docs/UserGuide.md:30]: They're kind of redundant, right? They show up under the command summary

Comment (zhongfu) [reply docs/UserGuide.md:30]: we also don't make use of these two features until the command summary

File: docs/UserGuide.md (hunk 3 above)
Comment (zhongfu) [reply docs/UserGuide.md:91]: ahhhhh

**Pull request merged**